automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/WorkspaceApiSpec.scala

```diff
@@ -39,6 +39,8 @@ class WorkspaceApiSpec extends FreeSpec with Matchers
           withWorkspace(projectName, prependUUID("writer-storage-cost"), aclEntries = List(AclEntry(writer.email, WorkspaceAccessLevel.Writer))) { workspaceName =>
             Orchestration.workspaces.waitForBucketReadAccess(projectName, workspaceName)(ownerAuthToken)
 
+            Thread.sleep(60 * 1000) // This was added as a workaround for https://broadworkbench.atlassian.net/browse/QA-1534
```

Comment:
`eventually` may be more appropriate to use here. You can tweak the timeout for it using the implicit `PatienceConfig` value

`OrchestrationApiSpec.scala` has some precedent for this, if you're curious